-- po/Makefile.in.in.orig       Tue May  2 18:37:39 2000
+++ po/Makefile.in.in   Tue May  2 18:37:57 2000
@@ -16,7 +16,7 @@
 top_srcdir = @top_srcdir@
 VPATH = @srcdir@

-prefix = @prefix@
+prefix = $(DESTDIR)/@prefix@
 exec_prefix = @exec_prefix@
 datadir = $(prefix)/@DATADIRNAME@
 localedir = $(datadir)/locale
